webapi/src/test/scala/org/knora/webapi/routing/RouteUtilZSpec.scala

```diff
@@ -0,0 +1,34 @@
+package org.knora.webapi.routing
+
+import zio.test.Assertion._
+import zio.test.ZIOSpecDefault
+import zio.test._
+import zio.test.assertTrue
+
+import dsp.errors.BadRequestException
+
+object RouteUtilZSpec extends ZIOSpecDefault {
+  val spec = suite("routeUtilZSpec")(
+    suite("function `urlDecode` should")(
+      test("given a valid encoding should return the decoded value") {
```

Comment:
```suggestion
      test("given a valid encoding, return the decoded value") {
```
...or remove the "should" from the suite